Production/python/customiseHLT.py

```diff
@@ -0,0 +1,327 @@
+      status = Var("status", int, doc='status word'),
+      longLived = Var("longLived", bool, doc='is long lived?'),
+      massConstraint = Var("massConstraint", bool, doc='do mass constraint?'),
+      # source: cmssw/PhysicsTools/NanoAOD/plugins/SimpleFlatTableProducerPlugins.cc 
+      trackDz = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.dz : -999.", float, doc = "track Dz"),
+      trackDxy = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.dxy : -999.", float, doc = "track Dxy"),
+      trackIsValid = Var("trackRef.isNonnull && trackRef.isAvailable", bool, doc = "track is valid"),
+      trackDzError = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.dzError : -999.", float, doc = "track DzError"),
+      trackDxyError = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.dxyError : -999.", float, doc = "track DxyError"),
+      trackPt = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.pt : -999.", float, doc = "track Pt"),
+      trackEta = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.eta : -999.", float, doc = "track Eta"),
+      trackPhi = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.phi : -999.", float, doc = "track Phi"),
+      trackPtError = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.ptError : -999.", float, doc = "track PtError"),
+      trackEtaError = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.etaError : -999.", float, doc = "track PtaError"),
+      trackPhiError = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.phiError : -999.", float, doc = "track PhiError"),
+      trackChi2 = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.chi2 : -999.", float, doc = "track Chi2"),
+      trackNdof = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.ndof : -999.", float, doc = "track Ndof"),
+      # source: DataFormats/TrackReco/interface/TrackBase.h
+      trackNumberOfValidHits = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.numberOfValidHits : -999.", float, doc = "number of valid hits found"),
+      trackNumberOfLostHits = Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.numberOfLostHits : -999.", float, doc = " number of cases where track crossed a layer without getting a hit."),
+      trackHitsValidFraction= Var("? trackRef.isNonnull && trackRef.isAvailable ? trackRef.validFraction : -999.", float, doc = "fraction of valid hits on the track"),
+      rawHcalEnergy = Var("rawHcalEnergy", float, doc='rawHcalEnergy'),
+      rawEcalEnergy = Var("rawEcalEnergy", float, doc='rawEcalEnergy'),
+      EcalEnergy = Var("ecalEnergy", float, doc='EcalEnergy'),
+      HcalEnergy = Var("hcalEnergy", float, doc='HcalEnergy'),
+    )
+  )
+  process.AK4PFJetsTable = cms.EDProducer("SimpleCandidateFlatTableProducer",
+    src = cms.InputTag( "hltAK4PFJets" ),
+    cut = cms.string(""),
+    name= cms.string("AK4PFJets"),
```

Comment:
```suggestion
    name= cms.string("Jet"),
```
(to be compatible with standard nano notation)